web_src/less/_repository.less

```diff
@@ -861,6 +861,10 @@
           }
         }
 
+        .inline-timeline-avatar {
+          display: none;
```

Comment:
Here and below, maybe there could be some comments about why the element should be hidden and what has been hidden. Otherwise in future developers may be confused about how to deal with it.